src/v2/Apps/Order/Routes/__tests__/Offer.jest.tsx

```diff
@@ -183,7 +199,14 @@ describe("Offer InitialMutation", () => {
       page = await buildPage()
     })
 
+    it("doesn't let the user continue if they haven't clicked any option", async () => {
+      await page.clickSubmit()
+      expect(mutations.mockFetch).not.toHaveBeenCalled()
+      expect(page.offerInput.text()).toMatch("Offer amount missing or invalid.")
+    })
+
     it("doesn't let the user continue if they haven't typed anything in", async () => {
+      page.selectCustomAmount()
```

Comment:
From now on the `OfferInput` is not visible until the user clicks on the custom option. So this first step is needed.